PhysicsTools/PatAlgos/python/recoLayer0/electronPFIsolationDepositsPAT_cff.py

```diff
@@ -1,14 +1,14 @@
 import FWCore.ParameterSet.Config as cms
 
-from CommonTools.ParticleFlow.Isolation.electronPFIsolationDepositsPFBRECO_cff import *
+import CommonTools.ParticleFlow.Isolation.electronPFIsolationDepositsPFBRECO_cff as m
```

Comment:
Yes. I agree that would be better. I will update the PR soon.

What I did was enough to keep unwanted things out of the Process object and the list of unscheduled modules. But you are right, the "m" symbol would show up in the symbol table of the importing modules, which is unnecessary and could cause problems if anyone tried to use it because of conflicts.